build/testing/integration.go

```diff
@@ -30,27 +36,57 @@ type testConfig struct {
 	token     string
 }
 
-func Integration(ctx context.Context, client *dagger.Client, base, flipt *dagger.Container) error {
+type testCaseFn func(_ context.Context, base, flipt *dagger.Container, conf testConfig) func() error
+
+func filterCases(caseNames ...string) (map[string]testCaseFn, error) {
+	if len(caseNames) == 0 {
+		return defaultCases, nil
+	}
+
+	for _, filter := range caseNames {
+		if _, ok := defaultCases[filter]; !ok {
+			return nil, fmt.Errorf("unexpected test case filter: %q", filter)
+		}
+	}
+
+	cases := map[string]testCaseFn{}
+	for n, fn := range defaultCases {
+		for _, filter := range caseNames {
+			if n == filter {
+				cases[n] = fn
+			}
+		}
+	}
```

Comment:
```suggestion
	for _, filter := range caseNames {
	      cases[filter] = defaultCases[filter]
	}
```

Would something like this work?

Reply:
Very much so. Great catch! 🙇